src/main/resources/db/migration/V6__create_application_table.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE `Application` (
+    Email varchar(255) NOT NULL,
+    RoleID INT NOT NULL,
+    S3Link varchar(255),
+    Status ENUM('Accepted', 'Rejected', 'In Progress'),
+    PRIMARY KEY (Email, RoleID),
+    FOREIGN KEY (Email) REFERENCES User(Email),
+    FOREIGN KEY (RoleID) REFERENCES Job_Roles(roleId)
+);
+
+INSERT INTO Application VALUES ('user@kainos.com', 1, 'https://academy-job-portal-cvs.s3.eu-west-1.amazonaws.com/the_extra_semicolon/TestDocument.docx', 'In Progress');
```

Comment:
Due to this being a public repo you have exposed the S3 link to the world. It's a security vulnerability and you will need to squash your commits. In this scenario you can just not insert anything into the table as this is the most secure way of having the code.